gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/infra/domain_service/documentation/ValidatePageSourceDomainServiceImpl.java

```diff
@@ -39,27 +47,22 @@ public class ValidatePageSourceDomainServiceImpl implements ValidatePageSourceDo
 
     private static final String GITHUB_SOURCE_TYPE = "github-fetcher";
     private static final String HTTP_SOURCE_TYPE = "http-fetcher";
-
     private static final String GITHUB_URL_PROPERTY = "githubUrl";
-
     private static final String FETCH_CRON_PROPERTY = "fetchCron";
-
+    private static final String VERSION_HEADER = "application/vnd.github.v3+json";
+    private static final String GITHUB_REPOSITORY_BRANCH_OR_TAG = "branchOrTag";
+    private static final String GITHUB_REPOSITORY_OWNER = "owner";
+    private static final String GITHUB_REPOSITORY = "repository";
+    private static final String GITHUB_FILEPATH = "filepath";
+    private static final String GITHUB_USERNAME = "username";
+    private static final String GITHUB_PERSONAL_ACCESS_TOKEN = "personalAccessToken";
```

Comment:
### :warning: Aqua detected sensitive data in your code  
**Category:** Code Generic   
**Description:** Code Generic Credential   
**Severity:** HIGH   
**Match:**     private static final String GITHUB_PERSONAL_ACCESS_TOKEN = "*******************";  
[This comment was created by Aqua Pipeline]